### ISSUE TITLE
`redshift.respones.py` does not provide `IamRoles` key in the response.

### ISSUE THREAD

**Issue by andharris** [spulec/moto#1487]
Redshift now allows `IamRoles` param so that specific roles can assume the cluster role and this is supported in `boto` but not able to be mocked in `moto`. 

I tried adding 
```
@mock_redshift
def test_create_cluster_with_iam_roles():
    iam_role = 'arn:aws:iam:::role/my-iam-role'
    client = boto3.client('redshift', region_name='us-east-1')
    cluster_id = 'my_cluster'
    client.create_cluster(
        ClusterIdentifier=cluster_id,
        NodeType="dw.hs1.xlarge",
        MasterUsername="username",
        MasterUserPassword="password",
        IamRoles=[iam_role],
    )
    response = client.describe_clusters(ClusterIdentifier=cluster_id)
    cluster = response['Clusters'][0]
    iam_roles = [role['IamRoleArn'] for role in cluster['IamRoles']]
    list(iam_roles).should.equal([iam_role.arn])
```

but get the following error
```
ERROR: test_redshift.test_create_cluster_with_iam_roles
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/venv/lib/python3.6/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/moto/moto/core/models.py", line 71, in wrapper
    result = func(*args, **kwargs)
  File "/moto/tests/test_redshift/test_redshift.py", line 311, in test_create_cluster_with_iam_roles
    iam_roles = [role['IamRoleArn'] for role in cluster['IamRoles']]
KeyError: 'IamRoles'
```

I've attempted a PR but the tests are still failing. Any input/guidance would be awesome and I'm happy to continue working on the PR to support this. 
https://github.com/spulec/moto/pull/1488

**Comment by bpandola** [comment]
Make sure you update the `to_json()` method in the `Cluster` class in `redshift/models.py` to include the list of IAM roles.

**Comment by andharris** [comment]
The PR is passing the tests now.

**Comment by spulec** [comment]
Closed with #1488
